src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/P11Util.java

```diff
@@ -23,6 +23,12 @@
  * questions.
  */
 
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2022, 2024 All Rights Reserved
+ * ===========================================================================
+ */
```

Comment:
It's not appropriate to add an IBM copyright notice when nothing here was authored by IBM.